subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -288,13 +288,65 @@ You can also assign collections or arrays of dependency notations to a configura
 [[sec:inspecting_dependencies]]
 === Inspecting dependencies
 
-You can generate dependency reports from the command line (see <<sec:listing_dependencies>>). With the help of the Project report plugin (see <<project_report_plugin>>) such a report can be created by your build.
+Gradle provides sufficient tooling to navigate large dependency graphs and mitigate situations that can lead to link:https://en.wikipedia.org/wiki/Dependency_hell[dependency hell]. Users can chose to render the full graph of dependencies as well as identify the selection reason and origin for a dependency. Gradle offers both capabilities as command line tooling and through visual representation via build scans.
 
-Since Gradle 1.2 there is also a new programmatic API to access the resolved dependency information. The dependency reports (see the previous paragraph) are using this API under the covers. The API lets you walk the resolved dependency graph and provides information about the dependencies. In future releases the API will grow to provide more information about the resolution result. For more information about the API please refer to the Javadocs on api:org.gradle.api.artifacts.ResolvableDependencies#getResolutionResult[]. Potential usages of the api:org.gradle.api.artifacts.result.ResolutionResult[] API:
+[[sec:listing_dependencies]]
+==== Listing all dependencies in a project
 
-* Creation of advanced dependency reports tailored to your use case.
-* Enabling the build logic to make decisions based on the content of the dependency graph.
+A project can declare one or more dependencies. Gradle can visualize the whole dependency tree for every configuration available in the project with the help of the _dependency report_.
```

Comment:
We talk a lot in this intro about the "dependency report" but then go on to mention build scans as well, making build scans feel quite secondary I think.

Reply:
I think it wouldn't feel right to first talk about build scans and then the dependency report. The dependency report _is_ the primary feature that ships with the Gradle distribution. The build scan is a nice addition but doesn't really trump the features users already have on that front (at least at the moment). Happy for you convince me otherwise.